setup.py

```diff
@@ -28,6 +48,12 @@
     url="https://github.com/cryptoadvance/specter-desktop",
     packages=find_namespace_packages("src", include=["cryptoadvance.*"]),
     package_dir={"": "src"},
+    package_data={
+        "": [
+            "src/cryptoadvance/specter/translations/*/*.mo",
+            "src/cryptoadvance/specter/translations/*/*.po",
+        ]
+    },
```

Comment:
When I run `setup.py install` it does the message compilation but when I look in the `build` dir the `translations` dir is missing. I tweaked `setup.py` to get it to include them:
```
    package_data={
        "": [
            "translations/*/LC_MESSAGES/messages.mo",
            "translations/*/LC_MESSAGES/messages.po",
        ]
    },
```
The `package_data` is relative to the `cryptoadvance.*` declared above.

That being said, I don't believe the *.po files are necessary for the build at this stage, right?

Reply:
Thank you, very essential point. I checked further and realised, that the files also don't end up in the pip/tar-package we're creating/uploading. The reason is that the package_data stuff is not integrated in a sdist, but only in a [bdist](https://stackoverflow.com/questions/7522250/how-to-include-package-data-with-setuptools-distutils) but we're currently creating sdist-binaries only. 
So the solution mentioned there was including it in the `MANIFEST.in` which i did as well.

I'm not 100% sure about the po-files. I think we should imho merge this PR and create a new pre-release testing this.